<launch>
	<group ns="camera">
		<node pkg="libuvc_camera" type="camera_node" name="mycam" output="screen" launch-prefix="gdb -ex run --args">
			<!-- Parameters used to find the camera -->
			<param name="vendor" value="0x2b03"/>
			<param name="product" value="0xf580"/>
			<param name="serial" value=""/>
			<!-- If the above parameters aren't unique, choose the first match: -->
			<param name="index" value="0"/>

			<!-- Image size and type -->
			<param name="width" value="1344"/>
			<param name="height" value="376"/>
			<!-- choose whichever uncompressed format the camera supports: -->
			<param name="video_mode" value="yuyv"/> <!-- or yuyv/nv12/jpeg -->
			<param name="frame_rate" value="100"/>

			<param name="camera_match_url" value="$(find libuvc_camera)/params/camera_match.yaml"/>
			<param name="camera_info_url" value="file://$(find libuvc_camera)/params/zed/left_camera_calibration_"/>
			<param name="camera_info_url_right" value="file://$(find libuvc_camera)/params/zed/right_camera_calibration_"/>
		</node>
	</group>
</launch>
